tensorflow/python/ops/histogram_ops.py

```diff
@@ -124,6 +128,7 @@ def histogram_fixed_width(values,
     TypeError: If any unsupported dtype is provided.
     tf.errors.InvalidArgumentError: If value_range does not
         satisfy value_range[0] < value_range[1].
+    ValueError: If the value of nbins is negative.
```

Comment:
The C++ implementation return `InvalidArgumentError`.

Reply:
Changed it to `tf.errors.InvalidArgumentError`